ociclient/cache/cache.go

```diff
@@ -46,11 +50,36 @@ func NewCache(log logr.Logger, options ...Option) (Cache, error) {
 		overlay = memoryfs.New()
 	}
 
+	//initialize metrics
+	metrics.CachedItems.WithLabelValues(opts.BasePath).Set(0)
+	metrics.CacheDiskUsage.WithLabelValues(opts.BasePath).Set(0)
+	metrics.CacheHits.WithLabelValues(opts.BasePath).Add(0)
```

Comment:
does it makes sense to have a dedicated hit for the in memory cache?

Reply:
good point, makes totally sense with garbage collection in mind. I've added another vector counter.